example/lib/src/file_picker_demo.dart

```diff
@@ -33,11 +55,32 @@ class _FilePickerDemoState extends State<FilePickerDemo> {
         .addListener(() => _extension = _fileExtensionController.text);
   }
 
+  Widget _buildFilePickerResultsWidget(
+      {required int itemCount,
+      required Widget Function(BuildContext, int) itemBuilder}) {
+    final ScrollController scrollController = ScrollController();
+    return Container(
```

Comment:
This container is redundant? See https://dart.dev/tools/linter-rules/avoid_unnecessary_containers

```suggestion
    return SizedBox(
```